lib/src/brandable/custom_theme.dart

```diff
@@ -140,6 +145,24 @@ class CustomTheme extends StatefulWidget {
     _CustomTheme inherited = (context.dependOnInheritedWidgetOfExactType<_CustomTheme>());
     return inherited.data;
   }
+
+  static ThemeKey getThemeKeyFor({@required String name}) {
+    if (name == ThemeKey.system.stringValue) {
+      return ThemeKey.system;
+    } else if (name == ThemeKey.light.stringValue) {
+      return ThemeKey.light;
+    } else {
+      return ThemeKey.dark;
+    }
+  }
+
+  static ThemeKey get systemThemeKey {
+    final brightness = WidgetsBinding.instance.window.platformBrightness;
+    if (brightness == Brightness.light) {
+      return ThemeKey.light;
+    }
+    return ThemeKey.dark;
```

Comment:
```suggestion
   return brightness == Brightness.light ? ThemeKey.light : ThemeKey.dark;
```
This could be shorten :)

Reply:
True!